configs/server/config.js

```diff
@@ -153,9 +159,9 @@ module.exports =
 	// if you use encrypted private key the set the passphrase
 	tls          :
 	{
-		cert : `${__dirname}/../certs/cert.pem`,
+		cert : `${__dirname}/../../certs/cert.pem`,
 		// passphrase: 'key_password'
-		key  : `${__dirname}/../certs/privkey.pem`
+		key  : `${__dirname}/../../certs/privkey.pem`
```

Comment:
Cert and privkey location is not changed so I think you should revert this change.

Reply:
Reverted